### PR TITLE
Connect GraphQL to the client

### PR DESCRIPTION
**Pull request by sioanis**
**Acceptance Criteria:**
- [x] Connect to server
- [x] Resolve cors issues

### CODE REVIEW

File: server/src/lambda/graphql.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const dataSources = () => ({
 });
 
 const server = new ApolloServer({
+  cors: { origin: '*', credentials: true },
```
Comment (sioanis) [inline server/src/lambda/graphql.js]: This is interesting, still debugging.

File: client/public/index.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
-      name="tripster"
+      name="description"
```
Comment (sioanis) [inline client/public/index.html]: Look at dose statz 👀 

Screen Shot 2020-01-19 at 8 40 06 PM

Comment (michaelyuen) [reply client/public/index.html]: but material is still trash

Comment (sioanis) [reply client/public/index.html]: 💩

**Pull request merged**